lms/templates/provider/authorize.html

```diff
@@ -1,59 +1,66 @@
-{% extends "main_django.html" %}
+## mako
```

Comment:
I ask because we should pass ``disable_courseware_js: True`` to it as otherwise we'll get the heavyweight version of the page from main.html.